chart/infra-server/Chart.yaml

```diff
@@ -8,3 +8,4 @@ sources:
   - https://github.com/stackrox/infra
 annotations:
   acsDemoVersion: 4.2.2
+  ocpCredentialsMode: Passthrough
```

Comment:
Even though I approved the PR...
Why is `ocpCredentialsMode` a chart annotation and not a value in one of the secret configuration or additional values files? 

The reason `acsDemoVersion` is an annotation is that
1) it describes the infra server behaviour (different demo version => different infra server behavior)
2) frequently changes & needs history
3) can be updated by automation & reviewed

Reply:
we could start a `values.yaml` that is checked into the repo and also put acsDemoVersion there.